symfit/core/fit_results.py

```diff
@@ -46,15 +47,20 @@ def __init__(self, model, popt, covariance_matrix, minimizer_output, minimizer,
 
     @property
     def iterations(self):
-        return self.minimizer_output.get('nit', None)
+        if 'nit' in self.minimizer_output:
+            return self.minimizer_output['nit']
+        elif 'niter' in self.minimizer_output:
+            return self.minimizer_output['niter']
```

Comment:
I really like this being a property, but I'd prefer to leave the translation to Minimizers. It's not super important though

Reply:
I agree that it would be better for all the minimizers to return something consistent in the future, but indeed this does not affect the API at the moment so I'm fine with this.

(p.s. for trust-constr it is suddenly called 'niter'... It's almost seems like a sport at scipy to be inconsistent.)

Reply:
I think it would be a fairly small change